docs/architecture/porting-existing-aspnet-apps/migrate-aspnet-core-2-1.md

```diff
@@ -0,0 +1,30 @@
+author: ardalis
+ms.date: 11/13/2020
+---
+
+# Migrating to ASP.NET Core 2.1
+
+ASP.NET Core 2.1 is an interesting release because it is the most recent supported .NET Core release to support both netcore and netframework runtimes. As such, it may offer an easier upgrade path for some applications when compared to upgrading all parts of the app to .NET Core at once. As an LTS release, support for .NET Core 2.1 will continue through August 2021.
+
+## Should applications run on .NET Framework with ASP.NET Core 2.1?
+
+Versions of .NET Core through 2.2 supported both .NET Core and .NET Framework runtimes. Does it make sense to migrate some or all of an application to ASP.NET Core 2.1 as a stepping stone, before porting over completely to .NET Core? Apps, or subsets of apps, could see their front-end ASP.NET logic ported to use ASP.NET Core, while still consuming .NET Framework libraries for business logic and infrastructure consumption. This approach may make sense when there is a relatively thin UI layer without much business logic, and a much larger set of functionality in class libraries.
+
+The main benefit of porting just the front end web layer to ASP.NET Core 2.1 is that the existing .NET class libraries can remain as is during the initial migration. They may be in continued use by other .NET applications or simply don't need to be in scope for the first iteration of a planned full migration to .NET Core. Reducing the scope of the initial migration for large applications helps provide incremental goals that act as stepping stones toward the desired end state, which is often a complete port to .NET Core.
+
+If you have an existing app that may use this strategy, some things you can do today to help prepare for the process are to move as much business logic, data access, and other non-UI logic out of the ASP.NET projects and into separate class libraries as possible. It will also help if you have automated test coverage of your system, so that you can verify that behavior remains consistent before and after the migration.
+
+If your app is so large that you can't migrate the entire web application at once, and you need to be able to deploy the new ASP.NET Core app side by side with the existing ASP.NET app, there are deployment strategies that can be used to achieve this. These are covered in [Chapter 5: Deployment Scenarios](deployment-scenarios.md).
+
+Keep in mind that ASP.NET Core 2.1 is the last LTS release of .NET Core that supports running on .NET Framework and consuming .NET Framework libraries. This release will be out of support soon, so if you require Microsoft support for your application, the window of opportunity for using this strategy won't be open much longer.
+
+## References
+
+- [Migrating from ASP.NET to ASP.NET Core 2.1](https://docs.microsoft.com/aspnet/core/migration/proper-to-2x/?view=aspnetcore-2.1)
```

Comment:
No need for the bulleted list, since there's only 1 item here.

```suggestion
[Migrating from ASP.NET to ASP.NET Core 2.1](https://docs.microsoft.com/aspnet/core/migration/proper-to-2x/?view=aspnetcore-2.1)
```